en/publications.md

```diff
@@ -7,8 +7,24 @@ permalink: /en/publications/
 share: false
 version: 1
 ---
+{%- if jekyll.environment == 'future' -%}
+- [Scaling book][]: a guide to effective practices and techniques
```

Comment:
I think we could add an introductory sentence above the bullet list. Something like "Optech publishes a variety of documentation and original material:"

Reply:
I tried this, but it feels redundant to me to say "Optech publishes [stuff]" under a big title that says "Publications".

Reply:
makes sense